packages/stylelint-plugin-mozaic/index.js

```diff
@@ -0,0 +1,34 @@
+const _ = require('lodash')
+const stylelint = require('stylelint')
+
+const checkCase = require('./lib/checkCase')
+const getSelector = require('./lib/getSelectors')
+const splitSelector = require('./lib/splitSelector')
+
+const ruleName = 'plugin/mozaic-bem-pattern'
+
+const isStringOrRegExp = x => {
+  return _.isString(x) || _.isRegExp(x)
+}
+
+module.exports = stylelint.createPlugin(
+  ruleName,
+  (options, secondaryOptionObject) => {
+    const allOptions = { ...options, ...secondaryOptionObject }
+    return (root, result) => {
+      if (!allOptions) return
+      const validOptions = stylelint.utils.validateOptions(result, ruleName, {
+        wordDelimiterStyle: [isStringOrRegExp],
+        modifierDelimiter: [_.isString],
```

Comment:
why do we check only modifierDelimiter ? does this still reflect the config ? 

```
wordDelimiterStyle: 'kebab-case',
delimiters: {
   modifier: '--',
   element: '__',
   media: '/@',
},
```

Reply:
let's call `delimiters` => `bemEntitiesDelimiters` and `wordDelimiterStyle` => `caseStyle` ? what do you think ?